docs/source/contributor-guide/roadmap.md

```diff
@@ -46,81 +46,12 @@ make review efficient and avoid surprises.
 
 # Quarterly Roadmap
 
-A quarterly roadmap will be published to give the DataFusion community
-visibility into the priorities of the projects contributors. This roadmap is not
-binding and we would welcome any/all contributions to help keep this list up to
-date.
+The DataFusion roadmap is driven the priorities of the contributors rather than
+any single organization or coordinating committee. We typically discuss roadmap
+approximately quarterly in GitHub issues.
 
-## 2023 Q4
```

Comment:
these items are quite old and I don't think they are serving much purpose anymore. Links to the more recent discussions I think are far more useful